libcontainer/cgroups/systemd/v1.go

```diff
@@ -207,6 +208,15 @@ func (m *legacyManager) Path(subsys string) string {
 	return m.paths[subsys]
 }
 
+func (m *legacyManager) Kill() error {
+	m.mu.Lock()
+	defer m.mu.Unlock()
+	// Since "cgroup.kill" is not a subsystem and we don't want to regenerate path using pid
+	// strip dir path from any available subsystem, most likely devices since its always there for v1
+	dirPath, _ := filepath.Split(m.paths["devices"])
```

Comment:
So, why are you using `filepath.Split` here?

Reply:
@kolyshkin We want path to tree where we are going to write  "cgroup.kill" for v1 , example `/sys/fs/cgroup/containerID/cgroup.kill`

Reply:
1. The path you provided as an example does not make sense for cgroup v1. 
2. The path that will be produced as a result of `filepath.Split` would be different from your example (it will actually be pointing to the parent cgroup, and I don't think it's a good idea to use it).

So, it looks like you're not testing this code at all (at least for v1). Do you?

Reply:
@kolyshkin ah my bad i didn't spent time testing this on `v1` but I followed the path construction which was being created on `crun` let me harden this for `v1` and if its not supported i will remove it. But if its not available for `v1` i think man-pages should be updated.

Reply:
@kolyshkin I am unable to write file `cgroup.kill` when working with `v1` under any of the paths even the base path but works when `cgroup_hierarchy=1` also tried with root.  But since its returning error for `v1` so code fallbacks to original slow process killing. PS: did not test this with `using v1 + hybrid`

Reply:
Now i am thinking should `v1` managers just return `error("Not supported")` but i am not sure if it works with  `v1+hybrid` or not.

Reply:
Todo: switch this with `m.paths[""]` to get the path for `v1+hybrid`. Wait for : #2087